### DISCUSSION TITLE
GameInput

### DISCUSSION THREAD

**Discussion by BMarques** [amerkoleci/Vortice.Windows#351] [General]
Hello,
With the release of Microsoft GDK June 2022, there's something called GameInput which is available publicly. Would it be technically possible to implement it in Vortice, despite not belonging to the Windows SDK? 
https://docs.microsoft.com/en-us/gaming/gdk/_content/gc/input/overviews/input-overview

**Comment by amerkoleci** [comment]
Yeah of course, I was never able to download GDK, are there any headers and dll's I can grab?

**Reply by BMarques** [reply]
https://github.com/microsoft/GDK

You need to download that repo and run the pgdk installer. It requires the VS c++ game development workload too.

**Reply by amerkoleci** [reply]
Will try it, does the licence permit dll/headers retested?

**Reply by BMarques** [reply]
Sorry, I have no idea :(
